keras_cv/layers/preprocessing3d/random_pasting_objects_test.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.layers.preprocessing3d.random_pasting_objects import RandomPastingObjects
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+OBJECT_POINT_CLOUDS = base_augmentation_layer_3d.OBJECT_POINT_CLOUDS
+OBJECT_BOUNDING_BOXES = base_augmentation_layer_3d.OBJECT_BOUNDING_BOXES
+
+
+class RandomPastingObjectsTest(tf.test.TestCase):
+    def test_augment_point_clouds_and_bounding_boxes(self):
+        add_layer = RandomPastingObjects(
+            label_index=1,
+            min_paste_bounding_boxes=1,
+            max_paste_bounding_boxes=1,
+        )
+        point_clouds = np.array(
```

Comment:
add comment here for the numerics? maybe say `point_clouds is a array of points for [center, far_left, near_east]`, etc

Reply:
Done.